codechain/config/mod.rs

```diff
@@ -112,24 +112,33 @@ impl Config {
     pub fn network_config(&self) -> Result<NetworkConfig, String> {
         debug_assert!(!self.network.disable.unwrap());
 
-        fn make_ipaddr_list(list_path: Option<&String>, list_name: &str) -> Result<Vec<IpAddr>, String> {
+        fn make_ipaddr_list(
+            list_path: Option<&String>,
+            list_name: &str,
+        ) -> Result<Vec<(IpAddr, Option<String>)>, String> {
```

Comment:
Why do you distinguish the case that there is a comment or not?

Reply:
If there's no tag on an IP address, how would you store it?

Reply:
An empty string.
IMO, there is no difference between ```8.8.8.8``` and ```8.8.8.8 # ```.

Reply:
I thought the empty string might not be a good idea at that time. I'll think about it after fixing some issues on sync.

Reply:
Changed from `Option<String>` to `String`